test/zmumu/tp_from_aod_MC.py

```diff
@@ -475,7 +477,7 @@
 
 if True: # turn on for tracking efficiency using L1 seeds
     process.probeL1 = cms.EDFilter("CandViewSelector",
-        src = cms.InputTag("l1extraParticles"),
+        src = cms.InputTag("gmtStage2Digis:Muon:HLT2"),
```

Comment:
Not really, but Muon is, I guess because the producer does not produce only that and as CandViewSelector does not ask explicitly for a type ... Anyhow I've changed this